### PR TITLE
kubernetes-cli: linux support

### PR DESCRIPTION
**Pull request by sublimino**

### CODE REVIEW

File: Library/Formula/kubernetes-cli.rb [hunk 1]
```diff
@@ -15,11 +15,12 @@ class KubernetesCli < Formula
   depends_on "go" => :build
 
   def install
+    os = if OS.linux? then 'linux' else 'darwin' end
```
Comment (sjackman) [inline Library/Formula/kubernetes-cli.rb]: Please use `?` and `:` and double quotes, like the line below. See the output of `brew audit --strict kubernetes-cli`

**Comment by sublimino** [comment]
I get exit code 0 from `brew audit --strict kubernetes-cli`, but the `brew tests` suite also fails for me before and after this change so it's probably something local. Could you paste the output if I need to see it please?

**Comment by sjackman** [comment]
`brew audit --strict kubernetes-cli` should have suggested using `?:` rather than `if then else end`. It looks good otherwise.

**Comment by sjackman** [comment]
The preferred commit log format is
`kubernetes-cli 1.0.0: Fix for Linuxbrew`

**Comment by sjackman** [comment]
Thanks for your contribution, Andrew.

**Pull request closed without merging**